src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -227,24 +227,23 @@ public function storeOutput()
         ) {
             $outputFilesConfig = $this->storageConfig["output"]["files"];
         }
-
+        if (isset($this->storageConfig["output"]["table_files"]) &&
+            count($this->storageConfig["output"]["table_files"])
+        ) {
+            $outputTableFilesConfig = $this->storageConfig["output"]["table_files"];
+        }
         $this->logger->debug("Uploading output tables and files.");
 
         $uploadTablesOptions = ["mapping" => $outputTablesConfig];
 
         $systemMetadata = [
             TableWriter::SYSTEM_KEY_COMPONENT_ID => $this->component->getId(),
             TableWriter::SYSTEM_KEY_CONFIGURATION_ID => $this->configId,
+            TableWriter::SYSTEM_KEY_RUN_ID => $this->clientWrapper->getBasicClient()->getRunId(),
         ];
         if ($this->configRowId) {
             $systemMetadata[TableWriter::SYSTEM_KEY_CONFIGURATION_ROW_ID] = $this->configRowId;
         }
-        if ($this->clientWrapper->hasBranch()) {
-            $systemMetadata[TableWriter::SYSTEM_KEY_BRANCH_ID] = $this->clientWrapper->getBranchId();
-        }
```

Comment:
This seemed redundant to me since the branchId is used as tag prefix

Reply:
it's not redundant, because this applies to both tables and files and it is very important for tables (to be able to distinguish those created in branch vs. those that only looks as if they're created in branch)

Reply:
omg, of course, thanks